lib/lightning_web/live/user_live/common_components.ex

```diff
@@ -0,0 +1,113 @@
+defmodule LightningWeb.UserLive.CommonComponents do
```

Comment:
Same question for the naming here ? Again, I really think that `CommonComponents` is a great name, I am just wondering about module naming consistency.

Reply:
done